backend/src/firebase/strategy/jwt.strategy.ts

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable prettier/prettier */
+import { Injectable } from "@nestjs/common";
+import { PassportStrategy } from "@nestjs/passport";
+import { Strategy, ExtractJwt } from 'passport-jwt'
+import * as dotenv from 'dotenv'
+
+dotenv.config()
+
+@Injectable()
+export class JwtStrategy extends PassportStrategy(Strategy, 'jwt'){
+  constructor(){
+    super({
+      jwtFromRequest: ExtractJwt.fromAuthHeaderAsBearerToken(),
+      ignoreExpirations: true,
+      secretOrKey: process.env.JWT_SECRET_KEY
+    })
+  }
+
+  async validate(payload: any){
```

Comment:
Nie używamy typu "any", any to jest w zasadzie workaround, który możesz zastosować chwilowo jak sobie nad czymś pracujesz, ale docelowo nie ma być żadnego "any" w aplikacji.